cmake/cmake_test/templates/test_CMakeLists.txt.in

```diff
@@ -28,7 +34,8 @@ include(cmakepp_lang/cmakepp_lang)
 cpp_set_global("CT_DEBUG_MODE" "@ct_debug_mode@")
 
 include("cmake_test/cmake_test")
+# Question: Why is this set separately from above?
```

Comment:
I believe this was done because of the `include("cmake_test/cmake_test")`, enabling `ct_debug_mode` alone would enable the assertions within CMakeTest while this line would reset the value to what the test expects

Reply:
Sounds good to me.